packages/components/src/components/CountrySelect/components/CountryList.ts

```diff
@@ -0,0 +1,252 @@
+export const CountryList: Record<string, string> = {
```

Comment:
hier bräuchten wir noch eine Alternative mit englischen namen, du kannst zb in der AlertIcon Component gucken, wie die Übersetzungen da gebaut sind